library/src/main/java/com/prolificinteractive/materialcalendarview/DayViewDecorator.java

```diff
@@ -0,0 +1,12 @@
+package com.prolificinteractive.materialcalendarview;
+
+import android.content.Context;
+
+/**
+ * Created by castrelo on 11/04/15.
+ */
+public interface DayViewDecorator {
+
+    public void decorate(DayView view, Context ctx);
```

Comment:
Context shouldn't need to be passed here; you can get access to Context from `DayView.getContext()`.